jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  verbose: true,
+  setupFiles: ["<rootDir>/jest.setup.js"],
+  coveragePathIgnorePatterns: ["jest.setup.js", ".story.js"],
+  testPathIgnorePatterns: ["src/Storyshots.test.js"]
```

Comment:
are you sure that this `testPathIgnorePatterns` property is working?

It seems that jest is trying to run the `Storyhots.test.js` file

Anyway, why would we ignore this file?

<img width="871" alt="screen shot 2017-10-09 at 15 57 35" src="https://user-images.githubusercontent.com/5287289/31353872-9a7915cc-ad0a-11e7-9c6c-07585ad47421.png">

Reply:
this import bug is a yarn / jest / babel thing, and the reason I've added ``yarn add -D jest`` on travis after all install steps. really strange and we might need to report this on the proper projects (probably on jest first)

if you add jest again it starts passing the test, and the most curious is that the lockfile dont change, remains the eexact same dependency tree